android/expoview/src/main/java/host/exp/exponent/experience/ExperienceActivity.java

```diff
@@ -400,8 +400,8 @@ protected void showOrReconfigureManagedAppSplashScreen(final JSONObject manifest
     }
 
     this.hideLoadingView();
-    ManagedAppSplashScreenConfiguration config = ManagedAppSplashScreenConfiguration.parseManifest(manifest);
     if (mManagedAppSplashScreenViewProvider == null) {
+      ManagedAppSplashScreenConfiguration config = ManagedAppSplashScreenConfiguration.parseManifest(manifest);
```

Comment:
what wsa the motivation for moving this?

Reply:
This `config` variable is only used in the `if` block, os it was just natural for me to scope it into `if` block as well 🤷

Reply:
ah, ok. i just wasn't sure if it was somehow intended to be related to the fix